test/functional/page_objects/visual_builder_page.ts

```diff
@@ -167,6 +167,34 @@ export function VisualBuilderPageProvider({ getService, getPageObjects }: FtrPro
       await element.click();
     }
 
+    /**
+     * setting label for markdown visualization
+     *
+     * @param {string} variableName
+     * @param type
+     * @memberof VisualBuilderPage
+     */
+    public async setMarkdownDataVariable(variableName: string, type: 'variable' | 'label') {
+      const SELECTOR = type === 'label' ? '[placeholder="Label"]' : '[placeholder="Variable name"]';
+      const prevRenderingCount = await PageObjects.visualize.getVisualizationRenderingCount();
+      if (variableName) {
+        await find.setValue(SELECTOR, variableName);
+      } else {
+        const input = await find.byCssSelector(SELECTOR);
+        if (process.platform === 'darwin') {
+          // Mac workaround
+          for (let i = 0; i <= type.length; i++) {
+            await input.pressKeys(browser.keys.BACK_SPACE);
+          }
+        } else {
+          await input.pressKeys([browser.keys.CONTROL, 'a']); // Select all for everything else
+          await input.pressKeys(browser.keys.NULL); // Release modifier keys
+          await input.pressKeys(browser.keys.BACK_SPACE); // Delete all content
+        }
```

Comment:
There is a new method available: `clearValueWithKeyboard(options = { charByChar: false })` Lets use it here